src/components/interfaces/MOBILE_API.xml

```diff
@@ -5808,6 +5809,12 @@
         <description> true if successful; false, if failed </description>
     </param>
   </function>
+
+  <function name="OnInteriorVehicleData" functionID="OnInteriorVehicleDataID" messagetype="notification">
+    <param name="moduleData" type="ModuleData">
```

Comment:
@AByzhynar we are using API from https://github.com/yang1070/sdl_core/blob/master/src/components/interfaces/MOBILE_API.xml
  ANd mandatory=true is there